src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1465,4 +1469,36 @@ public static MobEffect loadMobEffect(CompoundTag nbt, String key, @Nullable Mob
         }
     }
 
+    private static final ThreadLocal<RegistryResourceAccess> registryResouceAccess = new ThreadLocal<>();
+    private static final ThreadLocal<RegistryAccess> registryAccess = new ThreadLocal<>();
+
+    public static RegistryAccess.Writable createAndCacheWritable(ResourceManager manager) {
```

Comment:
Formatting: braces on newlines.
```suggestion
    public static RegistryAccess.Writable createAndCacheWritable(ResourceManager manager)
    {
```